src/app/qgsattributesformproperties.cpp

```diff
@@ -105,6 +105,13 @@ void QgsAttributesFormProperties::initAvailableWidgetsTree()
 
     item->setData( 0, FieldConfigRole, cfg );
     item->setData( 0, FieldNameRole, field.name() );
+
+    QString tooltip;
+    if ( !field.alias().isEmpty() )
+      tooltip = tr( "%1 (%2)" ).arg( field.name(), field.alias() );
```

Comment:
What's that supposed to translate?
Isn't that just going to send the string `"%1 (%2)"` to the QGIS transifex?

Reply:
Yep - don't brackets need translating too? Wouldn't right to left languages want to reverse this string?

Reply:
I don't have a compiler at hand, so can't check pure Qt, but look at this:

```py
dlg = QDialog()
label = QLabel()
dlg.setLayout(QGridLayout())
dlg.layout().addWidget(label)

a = 'לבצע'
b = 'תכנית'

lbl.setText('{} ({})'.format(a, b))
dlg.show
```

![screen shot 2018-08-13 at 21 41 51](https://user-images.githubusercontent.com/588407/44054020-b9a00846-9f41-11e8-8fac-37d82f7dbb17.png)

Reply:
What about the brackets themselves? Are they universal? I'm happy to use a fixed string here, I just thought I was being nice to translators :p

Reply:
I wouldn't say I'm an expert by any means, it's just a gut feeling that this might be more confusing and never actually have a translation.
@nirvn do you have an idea if this will help translators in some cases?

Reply:
It's hard to search for this. Only thing I've found so far is on https://m.mediawiki.org/wiki/Localisation

"Symbols, colons, brackets, etc. are parts of messages
Many symbols are localisable, too. Some scripts have other kinds of brackets than the Latin script has. A colon may not be appropriate after a label or input prompt in some languages. Having those symbols included in messages helps to better and less Anglo-centric translations, and by the way reduces code clutter."

Reply:
Interesting read.
I wonder if this applies to situations without context (like here) too or if it's more that one should prefer `tr('title: message')` over `tr('title') + ': ' + tr('message')`

Reply:
Colons are definitely a character that ideally needs localizing. For eg, in Khmer language, the colon is a different sign altogether.

As for the parenthesis, while most languages share the same character, it might not be bad practice to let translators be able to customize those too (possibly to add spaces if that's the local custom, changing ordering, etc).

Reply:
FWIW https://github.com/qgis/QGIS/blob/master/src/core/qgsmaplayermodel.cpp#L298 already has this same translated string (introduced April 2017)

Reply:
FWIW, it has never been translated...

Reply:
So.... how to proceed?

Reply:
Leave it and merge it, it's on the safe side, future proof and as long as we have no one with the required experience to give the final advice, we better stay on the safe side.

Reply:
OK, done.